modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/java/com/liferay/object/dynamic/data/mapping/form/field/type/internal/object/relationship/ObjectRelationshipDDMFormFieldTemplateContextContributor.java

```diff
@@ -194,24 +196,17 @@ private String _getLabelKey(DDMFormField ddmFormField) {
 			return labelKey;
 		}
 
-		ObjectDefinition objectDefinition = _getObjectDefinition(ddmFormField);
-
-		if ((objectDefinition != null) &&
-			(objectDefinition.getTitleObjectFieldId() > 0)) {
-
-			ObjectField objectField = _objectFieldLocalService.fetchObjectField(
-				objectDefinition.getTitleObjectFieldId());
+		ObjectField objectField = _getObjectField(ddmFormField);
 
-			if (objectField != null) {
-				String objectFieldName = objectField.getName();
+		if (objectField != null) {
```

Comment:
Hi @murilostodolni,
Could you do the pattern "Fail Fast/First"? Since the opposite condition would be simple:
`if (objectField == null) { return "id";}`

Reply:
When you do this refactor, could you squash the 2 commits into 1?

> The commits:
> [LPS-184908 Rename and simplify](https://github.com/liferay-objects/liferay-portal/pull/2192/commits/ade4696ca091e5d733eb3cdeb94fcf5e48aa7b19)
> [LPS-184908 Refactoring code to simplify](https://github.com/liferay-objects/liferay-portal/pull/2192/commits/a77e8f557fa048c5585e6df38d06a34e532a1883)

Since the two commit messages were generic and both commits were related (editing the same class), it is better to squash them. It was difficult for me to review them separated.

Reply:
Sorry for the confusion. I will refactor and squash the commits. Thank you @marcelabc